components/PageTitle/PageTitle.module.scss

```diff
@@ -1,18 +1,14 @@
 .background {
     @include full-width;
-
-    background: $color-base-50;
-    border-bottom: 1px solid $color-borders;
 }
 
 .container {
     @include container;
```

Comment:
Same here, since the block doesn't need to escape the main container anymore, we don't need the container mixin. So you can leave just the vertical padding in the styles.

Reply:
updated